velox/functions/prestosql/registration/JsonFunctionsRegistration.cpp

```diff
@@ -39,4 +40,42 @@ void registerJsonFunctions() {
   VELOX_REGISTER_VECTOR_FUNCTION(udf_json_parse, "json_parse");
 }
 
+void registerSIMDJsonFunctions() {
+  registerType("json", std::make_unique<const JsonTypeFactories>());
+
+  registerFunction<SIMDJsonArrayContainsFunction, bool, Json, bool>(
+      {"simd_json_array_contains"});
```

Comment:
We should register only one sets of functions and use Presto names for these. We can modify registerJsonFunctions to take useSimd boolean and use it to decide which set of functions to register.

Reply:
I use the Marco `USD_SIMDJSON` to decide which set of functions to register and USD_SIMDJSON=1 by default.

Reply:
Macros are not very convenient to use as they require recompiling. Can we use runtime flag instead?

Reply:
Done.